napari/components/chunk/_tests/test_chunk.py

```diff
@@ -0,0 +1,91 @@
+"""Tests for components.chunk."""
+
+import numpy as np
+import pytest
+
+from ....layers.image import Image
+from .. import ChunkKey, chunk_loader
```

Comment:
We use absolute imports in tests.

```suggestion
from napari.layers.image import Image
from napari.components.chunk import ChunkKey, chunk_loader
```

Reply:
Fixed, is there a reason tests do it differently? Is this in the Napari Style Guide?

Reply:
This will test the installed version, while in modules/packages you want to have a *consistent* version and that means relative imports. You could argue that you want to test a consistent version also, but generally tests are not necessarily in packages and relative imports are forbidden in non-packages.